src/ostree/ot-builtin-config.c

```diff
@@ -43,8 +43,16 @@ split_key_string (const char   *k,
                   char       **out_value,
                   GError     **error)
 {
+  const char *quotes_dot = strstr(k, "\".");
```

Comment:
Umm, so config file can have something like below:

[remote "org.mozilla.FirefoxRepo"]
gpg-verify=true
gpg-verify-summary=true
url=https://firefox-flatpak.mojefedora.cz/repo/
xa.title=Unofficial Firefox Flatpak Repository
xa.title-is-set=true

In some cases using strrchr won't work appropriately. For example, we want to fetch value of xa.title, in this case split_key_string() will have value of k as something like remote "org.mozilla.FirefoxRepo".xa.title . Using strrchr will give as match for title instead of xa.title .

Reply:
So looking at https://gitlab.gnome.org/GNOME/glib/blob/83a4cab12c2d00dbfe6013d071cff2da310109a4/glib/gkeyfile.c#L4119, it seems like group names can have almost any characters, except square brackets. If that's the case, then it seems like even `".` is not strong enough?

Here's a strawman: we support an alternative `[grouname].keyname` syntax. This will allow us to completely disambiguate them. E.g. one would write `ostree config get '[remote "foo.bar.baz"]'.xa.title`. What do you think?

(Note for remotes specifically, there's an open PR (https://github.com/ostreedev/ostree/pull/1166) that will make it more natural to change remote configs from the cmdline.)

Reply:
Yes! looking for [grouname].keyname sounds good to me, will update PR with changes. Does this change require updating ostree documentation anywhere?

Reply:
I think just the man page for the command (it's in `man/ostree-config.xml`).

Reply:
I like the --group idea, will update the PR with changes.